README.md

```diff
@@ -46,6 +46,50 @@ and are automatically released (multiple times a day, when needed) by
 Type checkers should be able to use these stub packages when installed. For more
 details, see the documentation for your type checker.
 
+### Stub package versioning
+
+Version numbers of third-party stub packages consist of at least four elements,
+where the last element represents the stub version. The other elements
+are the version of the package being stubbed.
```

Comment:
```suggestion
Version numbers of third-party stub packages consist of at least four parts.
All parts of the stub version, except for the last part, correspond to the
version of the runtime package being stubbed. For example, if the `types-foo`
package has version `1.1.2.0`, this signifies that the package contains stubs
for the `foo==1.1.2`. The final element of the version indicates that this is
the first revision of the stubs for `foo`: if an update to the stubs was
pushed the following day (but the stubs were still aiming to provide
annotations for `foo==1.1.2`), then the version of `types-foo` would
increment to `1.1.2.1`.
```

Reply:
I used a slightly modified version of your wording.